unuseable template - preact-typescript

Even with chai installed:

    npx create-snowpack-app --template @snowpack/app-template-preact-typescript unuseable-preact-typescript
    ...
    cd unuseable-preact-typescript
    npm i @types/chai
    ...
    npm run build
    ...
    [@snowpack/plugin-typescript] src/App.test.tsx(6,1): error TS2582: Cannot find name 'describe'. Do you need to install type definitions for a test runner? Try `npm i @types/jest` or `npm i @types/mocha`.
    [@snowpack/plugin-typescript] src/App.test.tsx(7,3): error TS2582: Cannot find name 'it'. Do you need to install type definitions for a test runner? Try `npm i @types/jest` or `npm i @types/mocha`.
    [@snowpack/plugin-typescript] Error: Command failed with exit code 2: tsc --noEmit
    src/App.test.tsx(6,1): error TS2582: Cannot find name 'describe'. Do you need to install type definitions for a test runner? Try `npm i @types/jest` or `npm i @types/mocha`.
    src/App.test.tsx(7,3): error TS2582: Cannot find name 'it'. Do you need to install type definitions for a test runner? Try `npm i @types/jest` or `npm i @types/mocha`.
    ...

--- a/create-snowpack-app/app-template-preact-typescript/package.json
+++ b/create-snowpack-app/app-template-preact-typescript/package.json
@@ -31,6 +31,8 @@
     "@snowpack/plugin-typescript": "^1.1.0",
     "@snowpack/web-test-runner-plugin": "^0.1.4",
     "@testing-library/preact": "^2.0.0",
+    "@types/chai": "^4.2.14",
+    "chai": "^4.2.0",
     "prettier": "^2.0.5",
     "snowpack": "^2.17.1",
     "typescript": "^4.0.3"

